### PR TITLE
#155738437 Android Setup

### PR DESCRIPTION
**Pull request by Rwothoromo**
#### What does this PR do?
Set up the repository well for Android

#### Description of Task to be completed?
- Add color palette
- Update `.gitignore`
- Set up IDE
- Update `README` App title and links to designs

#### How should this be manually tested?
Review the repository structure.

#### Any background context you want to provide?
The repository needed restructuring to suite an Android project.

#### What are the relevant pivotal tracker stories?
[#159051218](https://www.pivotaltracker.com/story/show/159051218)
[#155738437](https://www.pivotaltracker.com/story/show/155738437)
[#155738456](https://www.pivotaltracker.com/story/show/155738456)

### CODE REVIEW

File: .idea/codeStyles/Project.xml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+<component name="ProjectCodeStyleConfiguration">
```
Comment (idrisadetunmbi) [inline .idea/codeStyles/Project.xml]: this should go in `.gitignore`

File: .idea/runConfigurations.xml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (idrisadetunmbi) [inline .idea/runConfigurations.xml]: this should go in `.gitignore`
You can use a glob pattern to ignore this kind of files

File: .idea/vcs.xml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (idrisadetunmbi) [inline .idea/vcs.xml]: Same with this. ignore the `.idea` directory entirely

Comment (Rwothoromo) [reply .idea/vcs.xml]: I will make an update with the `.idea` ignored

**Pull request merged**